english/staypuft/ghostbusters/where-is-my-proton-pack.md

```diff
@@ -24,4 +30,4 @@ After so much excitement you decide to:
 
 [Watch Ghostbusters and see if you're in the credits?](watch-ghostbusters/watch-ghostbusters.md)
 
-[Jump to another Bill Murray movie](billMurrayMovies/billMurrayMovies.md)
```

Comment:
I'm not too concerned about this "modification". If you respond to the other comments, I'll be happy to merge this.